dist_tables/dml.rst

```diff
@@ -38,6 +38,105 @@ For example:
 
     If COPY fails to open a connection for a shard placement then it behaves in the same way as INSERT, namely to mark the placement(s) as inactive unless there are no more active placements. If any other failure occurs after connecting, the transaction is rolled back and thus no metadata changes are made.
 
+Inserting from a Query
+$$$$$$$$$$$$$$$$$$$$$$
+
+Applications like event data pipelines and real-time dashboards require fast queries across predefined aggregations of incoming data. One way to make these queries fast is by calculating and saving aggregates ahead of time. This is called "rolling up" the data and it avoids the cost of processing raw data at run-time. As an extra benefit, rolling up timeseries data into hourly or daily statistics can also save space. Old data may be deleted when its full details are no longer needed and aggregates suffice.
+
+For example, here is a table for tracking page views by url and a query to aggregate views per day.
+
+.. code-block:: postgresql
+
+  CREATE TABLE page_views (
+    site_id int,
+    url text,
+    host_ip inet,
+    view_time timestamp default now(),
+
+    PRIMARY KEY (site_id, url)
+  );
+
```

Comment:
(This has been fixed but the comment isn't disappearing because it is attached to a line in the code that hasn't changed.)